src/store/wallet/wallet-actions.ts

```diff
@@ -1,5 +1,7 @@
 import { createAction } from '@reduxjs/toolkit';
 
+import { TzProfile } from 'src/interfaces/tzProfile.interface';
+
 import { AccountInterface } from '../../interfaces/account.interface';
```

Comment:
The same as here: https://github.com/madfish-solutions/templewallet-mobile/pull/789/files#diff-dd81f14ebf9edab80c19a772a9ecac5f8f899eb83db3b120a9d029df011baef0R9